src/pod.rs

```diff
@@ -158,49 +174,45 @@ pub fn guarded_transmute_pod_pedantic<T: PodTransmutable>(bytes: &[u8]) -> Resul
 ///
 /// An error is raised in one of the following situations:
 ///
-/// - The data does not have a memory alignment compatible with `T`. You will
-///   have to make a copy anyway, or modify how the data was originally made.
 /// - The data does not have enough bytes for a single value `T`.
 ///
+/// # Safety
+/// 
+/// It is undefined behavior If the data does not have a memory alignment
+/// compatible with `T`. If this cannot be ensured, you will have to make a
+/// copy of the data, or change how it was originally made.
+///
 /// # Examples
 ///
 /// ```no_run
-/// # use safe_transmute::guarded_transmute_pod_many;
+/// use safe_transmute::pod::guarded_transmute_pod_many;
+/// use safe_transmute::SingleManyGuard;
 /// # include!("../tests/test_util/le_to_native.rs");
 /// # fn main() {
 /// // Little-endian
+/// unsafe {
 /// # /*
-/// assert_eq!(guarded_transmute_pod_many::<u16>(&[0x00, 0x01, 0x00, 0x02])?,
+///     assert_eq!(guarded_transmute_pod_many::<u16, SingleManyGuard>(&[0x00, 0x01, 0x00, 0x02])?,
 /// # */
-/// # assert_eq!(guarded_transmute_pod_many::<u16>(&[0x00, 0x01, 0x00, 0x02].le_to_native::<u16>()).unwrap(),
-///            &[0x0100, 0x0200]);
+/// #   assert_eq!(guarded_transmute_pod_many::<u16, SingleManyGuard>(&[0x00, 0x01, 0x00, 0x02].le_to_native::<u16>()).unwrap(),
+///                &[0x0100, 0x0200]);
+/// }
 /// # }
 /// ```
-pub fn guarded_transmute_pod_many<T: PodTransmutable>(bytes: &[u8]) -> Result<&[T], Error> {
-    check_alignment::<_, T>(bytes)?;
-    unsafe { guarded_transmute_many(bytes) }
+pub unsafe fn guarded_transmute_pod_many<T, G>(bytes: &[u8]) -> Result<&[T], Error>
```

Comment:
Requirements in brackets, as opposed to a `where` clause

Occurs on:
  * L202
  * L245
  * L293